### ISSUE TITLE
docs: missing use and run

### ISSUE THREAD

**Issue by jonschlinkert** [jonschlinkert/use#6]

**Comment by OnkelTem** [comment]
Agreed. 
7.1m of uses however. 
Indicating that everybody except me is telepath.
I even tried to run the examples but yet don't have a single idea what it does...

**Comment by cancerberoSgx** [comment]
Yes the description "Easily add plugin support to your node.js application." is more or less match what I'm needing. Unfortunately it's getting hard to even confirm that - the two examples are not clear / documented. You should better describe what is this all about, and what are you referring to when you say "plugins" if you want others to use this. Thanks
